README.md

```diff
@@ -15,6 +15,55 @@ Our documentation covers:
 * [Failures that Kani can spot](https://model-checking.github.io/kani/tutorial-kinds-of-failure.html)
 * [Kani's current limitations](https://model-checking.github.io/kani/limitations.html)
 
+## What guarantees does Kani provide?
+If Kani verifies your code under a proof harness, then, under the assumptions of the specification, you have assurance that all code reachable from the proof harness:
+* Does not have runtime panics 
+* All user-added assertions are correct.
+* There are no memory safety issues, including in unsafe code.
+
+Adding support for additional checks is prioritized based on customer need.
+[We would appreciate your feedback as to which are the highest priority areas for your projects](https://github.com/model-checking/kani/issues/new/choose).
+
+## What do I need to trust when running Kani?
+Kani analyzes the MIR representation of a Rust program.
+If the real-world behaviour of the program diverges from the MIR, then Kani's guarantees may not hold.
+Examples of how MIR can differ from real-world behaviour are discussed below. 
+
+Kani uses the program analysis tool [CBMC](https://github.com/diffblue/cbmc) as a back-end.
```

Comment:
Because as of now we only have one, I would say:
```suggestion
Kani uses the program analysis tool [CBMC](https://github.com/diffblue/cbmc) as its back-end.
```